ivy_tests/test_ivy/test_frontends/test_numpy/test_creation_routines/test_from_existing_data.py

```diff
@@ -39,3 +39,38 @@ def test_numpy_asarray(
         a=a,
         dtype=dtype,
     )
+
+
+# copy
+@handle_cmd_line_args
+@given(
+    dtype_and_a=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric"),
+        num_arrays=1,
+        min_num_dims=0,
+        max_num_dims=5,
+        min_dim_size=1,
+        max_dim_size=5,
+    ),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.numpy.copy"
+    ),
+)
+def test_numpy_copy(
+    dtype_and_a,
+    as_variable,
+    num_positional_args,
+    native_array,
+):
+    dtype, a = dtype_and_a
+    helpers.test_frontend_function(
+        input_dtypes=dtype,
+        as_variable_flags=as_variable,
+        with_out=False,
+        num_positional_args=num_positional_args,
+        native_array_flags=native_array,
+        frontend="numpy",
+        fn_tree="copy",
+        a=a,
+        dtype=dtype,
```

Comment:
there is no argument called "dtype" in the copy function, then why do you pass it in?

Reply:
@Daniel4078 totally agree. However, surprisingly it works when I pass `dtype=dtype`. If I dont pass it in, I got a library error :
`**Could not load dynamic library 'libcuda.so.1'; dlerror: libcuda.so.1: cannot open shared object file: No such file or directory**`

Reply:
ok then